docs/docs/migration-guide.mdx

```diff
@@ -20,17 +20,16 @@ components `DIETClassifier`, `ResponseSelector` and `TEDPolicy`. These include:
 2. The default loss function (`loss_type=cross_entropy`) can add an optional sigmoid cross-entropy loss of all similarity values to constrain
 them to an approximate range. You can turn on this option by setting `constrain_similarities=True`. This should help the models to perform better on real world test sets.
 
-Also, a new option `model_confidence` has been added to each ML component. It affects how a model's confidence for each label is computed during inference. It can take one of three values:
+Also, a new option `model_confidence` has been added to `DIETClassifier` and `ResponseSelector`. It affects how the model's confidence for each label is computed during inference. It can take one of two values:
 1. `softmax` - Similarities between input and label embeddings are post-processed with a softmax function, as a result of which confidence for all labels sum up to 1.
-2. `cosine` - Cosine similarity between input and label embeddings. Confidence for each label will be in the range `[-1,1]`.
-3. `inner` - Dot product similarity between input and label embeddings. Confidence for each label will be in an unbounded range.
-The default value is `softmax`, but we recommend using `cosine` as that will be the new default value from Rasa Open Source 3.0.0 onwards.
-The value of this option does not affect how confidences are computed for entity predictions in `DIETClassifier` and `TEDPolicy`.
+2. `inner` - Dot product similarity between input and label embeddings. Confidence for each label will be in an unbounded range.
```

Comment:
I've removed `cosine` as an option here so that any user coming to the migration guide from version `2.2`, does not see it as an option altogether. Is that okay?

Reply:
yeah i think that makes sense